src/components/header/headerNavbar.tsx

```diff
@@ -1,23 +1,25 @@
-import React from "react";
 import logo from "../../assets/GHGA_logo_clean.png";
-import { Navbar, Nav, Button, Row, Col } from "react-bootstrap";
+import { Navbar, Nav, Button, Row, Col, Offcanvas } from "react-bootstrap";
 import { NavLink } from "react-router-dom";
 import HeaderSearchbar from "./headerSearchbar";
+import { useState } from "react";
 
 const HeaderNavbar = () => {
-  const activePageStyle =
-    "btn btn-secondary p-0 h-100 m-0 mx-1 mx-xl-2 px-0 px-xl-2 pt-1 text-white";
-  const inactivePageStyle =
-    "btn btn-primary p-0 h-100 m-0 mx-1 mx-xl-2 px-0 px-xl-2 pt-1 text-white";
+  const navLinkClasses =
+    "h-100 m-0 mx-sm-1 mx-xl-2 py-2 pt-sm-1 pb-sm-0 px-sm-0 px-xl-2 w-100 w-sm-auto text-white btn";
+
+  const activePageStyle = navLinkClasses + " btn-secondary ";
+  const inactivePageStyle = navLinkClasses + " btn-primary";
+
+  const navColsSpanXS = "auto";
+  const navColsClasses = "text-center mb-2 mb-sm-0";
+
+  const [showOffCanvas, setShowOffCanvas] = useState(false);
+  const handleCloseOffCanvas = () => setShowOffCanvas(false);
+  const handleShowOffCanvas = () => setShowOffCanvas(true);
 
   return (
-    <Navbar
-      collapseOnSelect
-      expand="lg"
-      bg="primary"
-      variant="dark"
-      className="p-0 d-flex w-100"
-    >
+    <Navbar expand="lg" bg="primary" variant="dark" className="p-0">
       <Row className="w-100 mx-0 justify-content-between">
         <Col xs={12} sm={6} lg={3} className="text-center text-sm-start px-0">
```

Comment:
```suggestion
        <Col xs={12} sm={10} lg={3} className="text-center text-sm-start px-0">
```

To make it match the `sm={2}` below.